src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -30,6 +30,24 @@
       overflow: hidden;
       word-break: break-word;
       white-space: normal;
+
+      &.table__item--status {
+        .table__item--wrapper {
+          display: flex;
+          justify-content: space-between;
+        }
+      }
+
+      .zaak-hyperlink {
```

Comment:
Nice! Although instead of 'hyperlink' i'd just call this a '`contactmoment__link`' in BEM style, since this is more specific for contactmoment and not so much for Tables in this project and we decided not to use Dutch ('zaak') if possible + this link needs color-primary as a color. This way it will not interfere with the "contactmomenten__link" class + In future I'll be restyling this page dramatically, but if you could add these tiny changes then I will approve :-)